src/common/constants.ts

```diff
@@ -37,6 +43,29 @@ export const frameworkMap: Record<Framework['id'], Framework['name']> = {
   16: 'fxrecord',
 };
 
+export const supportedPerfdocsFrameworks: Record<
+  SupportedPerfdocsFramework,
+  string
+> = {
+  talos: 'talos',
+  awsy: 'awsy',
+  devtools: 'performance-tests-overview',
+};
+
+// TODO: remove these once the documentation for DevTools is complete
+export const nonDocumentedTestsDevTools = [
+  'reload-inspector:content-process',
+  'reload-inspector:parent-process',
+  'reload-debugger:content-process',
+  'reload-debugger:parent-process',
+  'reload-no-devtools:content-process',
+  'reload-no-devtools:parent-process',
+  'reload-netmonitor:content-process',
+  'reload-netmonitor:parent-process',
+  'reload-webconsole:parent-process',
+  'reload-webconsole:content-process',
+];
```

Comment:
@beatrice-acasandrei The array of `nonDumentedTestsDevTools` tests are present are exported from constants.